alf/networks/preprocessors.py

```diff
@@ -22,10 +22,11 @@
 import alf
 from alf.tensor_specs import TensorSpec, BoundedTensorSpec
 from alf.nest.utils import get_outer_rank
+from alf.networks.network import Network, SingletonInstanceNetwork
 import alf.utils.math_ops as math_ops
 
 
-class InputPreprocessor(nn.Module):
+class InputPreprocessor(Network):
```

Comment:
Since it's Network now, forward() should follow the interface for Network, i.e., return output, state

Reply:
Corrected.